src/python/WMCore/Services/ReqMgrAux/ReqMgrAux.py

```diff
@@ -411,7 +411,11 @@ def listDiskUsageOverThreshold(config, updateDB):
     if updateDB and not t0Flag:
         agentDrainMode = bool(len(overThresholdDisks))
         if agentConfig and (agentDrainMode != agentConfig["AgentDrainMode"]):
-            reqMgrAux.updateWMAgentConfig(config.Agent.hostName, {"AgentDrainMode": agentDrainMode},
+            if agentConfig["UserDrainMode"] == 'True':
```

Comment:
Looking into this again, it looks like these drain metrics are actually of the boolean type, e.g.:
https://cmsweb.cern.ch/reqmgr2/data/wmagentconfig/cmsgwms-submit13.fnal.gov

so I guess we should update this if statement to `if agentConfig["UserDrainMode"] is True:` or `if agentConfig["UserDrainMode"]:`

Perhaps you can check it in that testbed agent that you can play with (do you still have such agent?)?

Reply:
Hi Alan, 
Yes I have the vocms264 testbed agent.

I tested out the Pull Request and verified that it's working. I have attached the document with logs and issues I encountered, Please check it out and let me know if everything looks good.

Thank you.

[Hassan_PR_12142_Agent_Drain_Mode_Alert_Tesst.pdf](https://github.com/user-attachments/files/17377904/Hassan_PR_12142_Agent_Drain_Mode_Alert_Tesst.pdf)